tests/js/server/shell/shell-transactions.js

```diff
@@ -1256,7 +1225,6 @@ function transactionOperationsSuite () {
       }
 
       c2 = null;
-      internal.wait(0);
```

Comment:
didn't we add these to trigger a garbage collection? why aren't they needed anymore?

Reply:
As far as I know, this garbage collection was only useful in the olden days of MMFiles.